src/components/forms/NestedDropdownArea.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import Dropdown from './Dropdown'
+import ParentDropdown from './ParentDropdown'
+import "../../styles/form.css"
+
+export default class NestedDropdownArea extends Component {
+  render() {
+    const { options, setFieldState, defaultValue, name, state } = this.props
+    let nestedDropdownField = options.filter( option => {
```

Comment:
Whitespace is weird inside filter function. Also, we can use `find` instead of `filter` to find the first entry, which means we would no longer need to get the `[0]` entry. The identifier `state` is **very** confusing here, since state is a keyword that is very meaningful in React, please rename. Finally, I'd refactor this into a separate, meaningfully named function:
```jsx
nestedField() {
  const { options, state, name } = this.props

  return options.find( option => option.value === state[ name ] ).field
} 
```